src/parsers/js/transformers/babel6/index.js

```diff
@@ -26,7 +27,7 @@ export default {
     );
 
     return babel.transform(code, {
-      presets: [],
+      presets: [presets[3]],
```

Comment:
I guess you rather want to put it into a separate property in the `loadTransformer` and use from there; it doesn't make sense to double-include it in the generic array for the `compileModule` anyway.